metadata/nz_building_outlines_lifecycle.xml

```diff
@@ -99,21 +99,21 @@ This table provides the relationships between building outlines within mainland
 A building outline is a 2D representation of the roof outline of a building which has been classified from LINZ aerial imagery using a combination of automated and manual processes to extract and refine a building roof outline.
 
 Data vintage
-This dataset does not include building outline geometry, but provides the linkages in building outline id's from different aerial imagery sources in the [NZ Building Outlines Full History] (https://data.linz.govt.nz/layer/xxxxx) dataset when building outlines change from single outlines split into multiple outlines, and from multiple outlines merged as single outlines.
+This dataset does not include building outline geometry, but provides the linkages between building_id's from different aerial imagery sources in the [NZ Building Outlines All Sources] (https://data.linz.govt.nz/layer/101292) dataset when building outlines change from single outlines split into multiple outlines, and from multiple outlines merged as single outlines.
 
-This dataset will be updated with changes in building id relationships as new aerial imagery becomes available.
+This dataset will be updated with changes in building_id relationships as new aerial imagery becomes available.
 Please refer to the [NZ Building Outlines Data Dictionary] (https://nz-buildings.readthedocs.io/en/latest/introduction.html) for detailed metadata and information about this dataset.
 
 **Related building outlines data**
 
-[NZ Building Outlines Full History] (https://data.linz.govt.nz/layer/xxxxx) - directly related to this dataset, it contains all combinations of building outlines from multiple years of imagery that have existed since the beginning of this dataset.
+[NZ Building Outlines All Sources] (https://data.linz.govt.nz/layer/101292) - directly related to this dataset, it contains all combinations of building outlines from multiple years of imagery that have existed since the beginning of this dataset.

```

Comment:
```suggestion
[NZ Building Outlines (All Sources)] (https://data.linz.govt.nz/layer/101292) - directly related to this dataset, it contains all combinations of building outlines from multiple years of imagery that have existed since the beginning of this dataset.
```